Sources/AblyAssetTrackingPublisher/DefaultPublisher.swift

```diff
@@ -703,11 +715,16 @@ extension DefaultPublisher {
     }
 
     private func resolveResolution(trackable: Trackable) {
-        let currentRequests = requests[trackable]?.values
-        let resolutionSet: Set<Resolution> = currentRequests == nil ? [] : Set(currentRequests!)
-        let request = TrackableResolutionRequest(trackable: trackable, remoteRequests: resolutionSet)
-        resolutions[trackable] = resolutionPolicy.resolve(request: request)
-        enqueue(event: ChangeLocationEngineResolutionEvent())
+        /**
+         Re-calculate resolution if `constantLocationEngineResolution` is not defined
+         */
+        if constantLocationEngineResolution == nil {
+            let currentRequests = requests[trackable]?.values
+            let resolutionSet: Set<Resolution> = currentRequests == nil ? [] : Set(currentRequests!)
+            let request = TrackableResolutionRequest(trackable: trackable, remoteRequests: resolutionSet)
+            resolutions[trackable] = resolutionPolicy.resolve(request: request)
+            enqueue(event: ChangeLocationEngineResolutionEvent())
+        }
```

Comment:
It looks like you've disabled the whole resolution mechanism instead of only the part about location engine :sweat_smile: If the `constantLocationEngineResolution` is set then we should only disable the location engine resolution changes but the trackable resolutions should still be calculated and used :wink:

Reply:
🤦‍♂️

chasnged here 552c0290d523e7e0d8c8800cb692393e21b7d67d